public/src/components/PostItem/CommentComponent.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faAngleUp,
+    faAngleDown,
+    faCommentAlt,
+} from '@fortawesome/free-solid-svg-icons';
+import methods from '../../utils/methods';
+
+const {
+    getDate,
+    reformatTextToHtml,
+} = methods;
+
+export default class CommentsComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.calculateDate = this.calculateDate.bind(this);
+        this.getCommentClassName = this.getCommentClassName.bind(this);
+        this.getOffsetLines = this.getOffsetLines.bind(this);
+    }
+
+    getCommentMessageHtml() {
+        const { body } = this.props;
+        return (
+            <div className="comment-body__text"
+                 dangerouslySetInnerHTML={{ __html: reformatTextToHtml(body) }}
+            />
+        );
+    }
+
+    getOffsetLines() {
+        const { depth } = this.props;
+        const elementsToReturn = [];
+        for (let i = 0; i < depth; i += 1) {
```

Comment:
I think you can try to use `Array.from({length: depth}, () => ...)` syntax instead https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

Reply:
fixed https://github.com/YuriyBC/Reddit-archive/commit/b4002e8d7bc8f4fe69cb0938571d9d83b2f5868d